lib/Models/WebMapServiceCatalogItem.ts

```diff
@@ -883,7 +913,8 @@ class WebMapServiceCatalogItem
         "width",
         "height",
         "bbox",
-        "layers"
+        "layers",
+        "version"
```

Comment:
This is here as a temporary fix until Cesium implements this fix
https://github.com/CesiumGS/cesium/issues/9021

I should probably annotate a To-DO to remove it in the future

Reply:
added todo